### PR TITLE
update ShoRAH 1.99.2 (and 1.99.0 & 1.1.3)

### PR DESCRIPTION
**Pull request by DrYak**
update ShoRAH 1.99.2 (replaces 1.99.1)
and fixup on 1.99.0 and 1.1.3

 - fixes HTSlib 1.10 crashes
 - tests !

Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by DrYak** [comment]
@BiocondaBot please add label

**Comment by DrYak** [comment]
@BiocondaBot please fetch artifacts

File: recipes/shorah/1.99.0/meta.yaml [hunk 1]
```diff
@@ -33,16 +35,23 @@ requirements:
     - gsl
     - zlib
     - blas {{ blas }}
+    # BUG ShoRAH version 1.99.0 crashes on Mac OS X if setuptools is absent *fixed in ShoRAH 1.99.1)
     - setuptools
```
Comment (bgruening) [inline recipes/shorah/1.99.0/meta.yaml:39]: You can only include this for osx

Comment (bgruening) [reply recipes/shorah/1.99.0/meta.yaml:39]: '# osx'

File: recipes/shorah/1.1.3/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 6
+  number: 7
```
Comment (bgruening) [inline recipes/shorah/1.1.3/meta.yaml:14]: If you prefere you can use a build.sh file or specify multiple commands in a list

Comment (DrYak) [reply recipes/shorah/1.1.3/meta.yaml:14]: So we have a guarantee that the command will be ran sequencially?

Okay, will do. Thanks for the suggestion.

**Comment by DrYak** [comment]
Suggestions merged-in, and successfully passed tests.

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**